yarn.lock

```diff
@@ -3259,6 +3261,11 @@ cross-spawn@^7.0.0, cross-spawn@^7.0.2, cross-spawn@^7.0.3:
     shebang-command "^2.0.0"
     which "^2.0.1"
 
+crypto@^1.0.1:
```

Comment:
Please remove, `crypto` is part of Nodejs std lib

Reply:
I think it's part of node v17, but where are on v16?